Modules/@babylonjs/react-native/android/build.gradle

```diff
@@ -74,6 +74,20 @@ android {
         ndk {
             abiFilters "arm64-v8a", "armeabi-v7a", "x86"
         }
+        packagingOptions {
+            // TODO if def based on gradle version where we only do this for 4.0.0+?
```

Comment:
we can probably remove this comment, libBabylonNative.so is the only lib we distribute and this exclude isn't included in the npm package build.gradle file. @ryantrem are we okay with having everyone build the libBabylonNative.so with gradle 4.1.0?

Reply:
I have 3.5 and this didn't break anything for me, so I think we could just update the comment to say that this is to support the Android gradle plugin 4.0.0+. Please call it "Android gradle plugin" though and not "gradle", because they are versioned differently. We are using gradle version 6.7.

Reply:
this should now be fixed